### PR TITLE
Flywheel tweaks

### PR DESCRIPTION
**Pull request by ptx2**

### CODE REVIEW

**Review by doudar — APPROVED** [review]
This looks awesome! 

All reviewed and I don't see any conflicts. 

Thanks for the help - I can't wait until we have time to implement our buffer to get the rest working. 

@kaaden , I'll let you merge this as I'm not certain if we want it going to master or develop. It would be an easy choice normally but we're in this rut again where develop is behind so it's a lot of file changes if I were to try to merge this in to develop.

**Comment by kadaan** [comment]
@ptx2 Thanks!

File: src/BLE_Client.cpp
Comment (kadaan) [inline src/BLE_Client.cpp:402]: Geez, this is getting nasty.  We need to solve this quick...

File: src/sensors/FlywheelData.cpp
Comment (kadaan) [inline src/sensors/FlywheelData.cpp:31]: Interesting...  Why is flywheel data so odd...

**Review by kadaan — COMMENTED** [review]
@ptx2 I assume this "works on your machine" and get you to the point where the data is overwritten too quickly for SS2K to act upon?

**Comment by ptx2** [comment]
> @ptx2 I assume this "works on your machine" and get you to the point where the data is overwritten too quickly for SS2K to act upon?

Exactly! I should note that the power/cadence decoding change is untested (because of the race) but I think it should be right.

File: src/BLE_Client.cpp
Comment (doudar) [reply src/BLE_Client.cpp:402]: To only get a lot worse quickly if we keep focusing on the BLE multiplexer aspect, which I'd like to.

**Comment by doudar** [comment]
After the latest merge on develop, this is now an easy fit into Develop. 

Party Time! 🥳

**Comment by doudar** [comment]
@kadaan , THis is ready to merge into Develop. I'm uncertain how to incorporate a new changlog into an existing PR though.

**Review by doudar — APPROVED** [review]

**Pull request merged**
